src/ballistica/scene/node/spaz_node.cc

```diff
@@ -3884,6 +3884,11 @@ void SpazNode::Step() {
     if (!holding_something_ && hold_node_.exists()) hold_node_.Clear();
   }
 
+  // If we're knocked out, stop any mid-progress pick up.
+  if (knockout_) {
+    holding_something_ = false;
```

Comment:
I've just got this working by replacing this line with 
```c++
DropHeldObject();
```
Think about whether it suits you.